services/site/src/pages/press-kit.tsx

```diff
@@ -215,33 +225,48 @@ const PressKit: React.FunctionComponent = () => {
             <h2 className={clsx("mb-8", "h4", "md:h3")}>Application Screenshots</h2>
             <div className="flex flex-wrap items-center">
               <div className="w-[70ch] h-full my-6 md:m-8">
-                <img
+                <Image
                   alt="Allyphant website: An overview of all available challenge topics and their difficulty."
-                  src="/images/showcase/a11yphant_challenge_overview.jpg"
+                  src={a11yphantWebsite}
+                  placeholder="blur"
+                  priority
```

Comment:
```suggestion
```

> Should only be used when the image is visible above the fold.
> \- https://nextjs.org/docs/pages/api-reference/components/image#priority

Since the images are far down on the page, it doesn't make sense to me to load them with priority. Users won't see them on first glance anyway